bazel/foreign_cc/luajit.patch

```diff
@@ -73,6 +73,12 @@ index 0000000..9c71271
 +    # fail on it.
 +    os.environ["LSAN_OPTIONS"] = "exitcode=0"
 +
++    # Blacklist LuaJIT from ASAN for now.
++    # TODO(htuch): Remove this when https://github.com/envoyproxy/envoy/issues/6084 is resolved.
++    with open("clang-asan-blacklist.txt", "w") as f:
```

Comment:
```
if "-fsanitize=address" in os.environ["TARGET_CFLAGS"]:
    ...
```

Reply:
Nup :) This is a quirk of how flags are handled by `rules_foreign_cc`, but I already tried that, might need to split configure/make, looking into simpler solution right now.

Reply:
You might just want to add this to top level of envoy, instead of here? and only pass -fsanitize-blacklist on ASAN builds.

Reply:
This isn't a great idea either, due to how paths are handled by Bazel. I will fix this later today, busy in meeting until then.